en/4-empower-staff-deliver-better-services.md

```diff
@@ -200,63 +160,24 @@ The technology you choose to build your service must help you respond quickly an
-- Contract allows us to use tools, services, and hosting from vendors with a variety of pricing models, including fixed fees and variable models like "pay-for-what-you-use" services **(Digital Services Playbook (US))**
-- Contract specifies a warranty period where defects uncovered by the public are addressed by the vendor at no additional cost to the government **(Digital Services Playbook (US))**
-- Contract includes a transition of services period and transition-out plan **(Digital Services Playbook (US))**
-- Uncover user needs and confirm with clients their understanding of the desired result. **(Do - Digital Design Playbook (ISED))**
-- Look for opportunities to integrate with other relevant or similar services (i.e. joined-up services) to simplify the client experience **(Do - Digital Design Playbook (ISED))**
-- Look for opportunities to use business intelligence and analytics (BI/BA) to improve the client experience. **(Do - Digital Design Playbook (ISED))**
-- Identify policy constraints. **(Do - Digital Design Playbook (ISED))**
-- Identify opportunities for improving processes and reducing complexity. **(Do - Digital Design Playbook (ISED))**
-- Improve the service first and consider if a policy change or update might be needed. Second, build the digital platform/website. **(Do - Digital Design Playbook (ISED))**
-
-**Make one person accountable for the service (formerly guideline 4.5):**
-{: .dpgn-data-ignore}
-
-- Make one person accountable for the service
-  - Has a product management background with technical experience to assess alternatives and weigh trade-offs
-  - Has a strong knowledge of the procurement process and is able to articulate value for money is achieved when buying any tools
-  - Has appropriate seniority, knowledge, skill and personal qualities
-  - Has agreement from all stakeholders about authority to assign tasks and make decisions about features and technical implementation details
-  - Utilizes streamlined procurement processes and utilize experienced budgeting and contracting resources when contracting out development work
-  - Has a work plan that includes budget estimates and identifies funding sources **(Digital Services Playbook (US))**
-  - Develops a plan and capabilities to track concurrent users in real-time, and monitor user behaviors in the aggregate to determine how well the service meets user needs
-  - Identifies data you need to capture, where you need to capture it from and how you'll capture it based on the projected size and shape of the service
-  - Develops an ongoing roadmap for performance analysis and designate responsibility for identifying actionable data insights
-  - Includes the ability to improve the service on a frequent basis through iteration enabled by the right capacity, resources, and technical flexibility
-
-**Build It Right Principles (GC):**
```

Comment:
The Build It Right Principles content is currently being used by a view. If you don't want it to be visible in the output of Standards 4 then apply <code>.dpgn-standard-hide</code> to the content rather than deleting it (so it still remains visible in the separate view).